app/workers/databases.php

```diff
@@ -321,7 +339,13 @@ protected function deleteIndex(Document $database, Document $collection, Documen
             $dbForProject->deleteDocument('indexes', $index->getId());
         } catch (\Throwable $th) {
             Console::error($th->getMessage());
-            $dbForProject->updateDocument('indexes', $index->getId(), $index->setAttribute('status', 'stuck'));
+            $dbForProject->updateDocument(
+                'indexes',
+                $index->getId(),
+                $index
+                ->setAttribute('status', 'stuck')
```

Comment:
```suggestion
                ->setAttribute('status', 'stuck')
```

What status types do we support? can we change this to error? cc @christyjacob4

Reply:
We currently support 
1. available
2. failed
3. stuck
4. deleting
5. processing

Reply:
considering this comment [here](https://github.com/appwrite/appwrite/blob/master/app/workers/databases.php#L148-#L153), i think changing it to `failed` can be misleading.

`failed` indicates the attribute was never created in the db, whereas `stuck`
indicates the attribute is available in the db but the deletion operation failed. Since the exception thrown [here](https://github.com/appwrite/appwrite/blob/master/app/workers/databases.php#L318) would only be of a case where the index is available in the db, I think `stuck` makes sense by the definition.  If we want to deprecate `stuck` and go with `failed` i think we should redefine `failed` and what it means before doing so.